discord/ext/converters/range.py

```diff
@@ -0,0 +1,13 @@
+from typing import Tuple
+
+from discord.ext import commands
+
+
+class InRangeConverter(commands.Converter):
+    def __init__(self, *args, **kwargs) -> None:
+        self.range = range(*args, **kwargs)
```

Comment:
`range` doesn't take any keyword arguments.

```suggestion
        self.range = range(*args)
```